api/sip/client.py

```diff
@@ -31,7 +31,10 @@
 import socket
 import ssl
 import tempfile
-from typing import Optional
+from ssl import _SSLMethod
```

Comment:
While I agree its preferable not to use underscore methods, it seems like there isn't much choice in some cases, especially for annotations. It seems like some types are marked as private, because they aren't meant to be instantiated directly, but but have a factory function create them.